### ISSUE TITLE
`WHO mask` no longer returns any result

### ISSUE THREAD

**Issue by progval** [ircd-hybrid/ircd-hybrid#39]
on 16882760aff6b736154a820b84e82a7677e2879e (31b284171a9ad508f41fd75ff221b820987db02c's parent), `WHO *` returns a list of users we share a channel with, as expected:

```
1661594302.433 1 -> S: NICK coolNick
1661594302.433 1 -> S: USER myusernam 0 * :My UniqueReal Name
[...]
1661594302.434 1 -> S: JOIN #chan
1661594302.434 S -> 1: :coolNick!~myusernam@localhost MODE coolNick :+i
1661594302.434 S -> 1: :coolNick!~myusernam@localhost JOIN :#chan
1661594302.434 S -> 1: :My.Little.Server MODE #chan +nt
1661594302.434 S -> 1: :My.Little.Server 353 coolNick = #chan :@coolNick
1661594302.434 S -> 1: :My.Little.Server 366 coolNick #chan :End of /NAMES list.

1661594302.434 2 -> S: NICK otherNick
1661594302.434 2 -> S: USER username * * :Realname
[...]
1661594302.436 S -> 2: :otherNick!~username@localhost MODE otherNick :+i
1661594302.436 2 -> S: JOIN #chan
1661594302.436 S -> 2: :otherNick!~username@localhost JOIN :#chan
1661594302.436 S -> 2: :My.Little.Server 353 otherNick = #chan :otherNick @coolNick
1661594302.436 S -> 2: :My.Little.Server 366 otherNick #chan :End of /NAMES list.

1661594302.436 2 -> S: WHO *
1661594302.436 S -> 2: :My.Little.Server 352 otherNick #chan ~username localhost My.Little.Server otherNick H :0 Realname
1661594302.436 S -> 2: :My.Little.Server 352 otherNick #chan ~myusernam localhost My.Little.Server coolNick H@ :0 My UniqueReal Name
1661594302.437 S -> 2: :My.Little.Server 315 otherNick * :End of /WHO list.
```

on 31b284171a9ad508f41fd75ff221b820987db02c, `WHO *` no longer returns anything:

```
1661594319.517 1 -> S: NICK coolNick
1661594319.517 1 -> S: USER myusernam 0 * :My UniqueReal Name
[...]
1661594319.518 1 -> S: JOIN #chan
1661594319.518 S -> 1: :coolNick!~myusernam@localhost MODE coolNick :+i
1661594319.518 S -> 1: :coolNick!~myusernam@localhost JOIN :#chan
1661594319.518 S -> 1: :My.Little.Server MODE #chan +nt
1661594319.518 S -> 1: :My.Little.Server 353 coolNick = #chan :@coolNick
1661594319.518 S -> 1: :My.Little.Server 366 coolNick #chan :End of /NAMES list.

1661594319.519 2 -> S: NICK otherNick
1661594319.519 2 -> S: USER username * * :Realname
[...]
1661594319.519 S -> 2: :otherNick!~username@localhost MODE otherNick :+i
1661594319.519 2 -> S: JOIN #chan
1661594319.520 2 -> S: PING synchronize626921.520337814
1661594319.520 S -> 2: :otherNick!~username@localhost JOIN :#chan
1661594319.520 S -> 2: :My.Little.Server 353 otherNick = #chan :otherNick @coolNick
1661594319.520 S -> 2: :My.Little.Server 366 otherNick #chan :End of /NAMES list.

1661594319.520 2 -> S: WHO *
1661594319.520 S -> 2: :My.Little.Server 315 otherNick * :End of /WHO list.
```

**Comment by miwob** [comment]
Should be fixed now in 752e2ac760367c19c730121a3562b86035912669. That was a stupid one. Thanks.

**Comment by progval** [comment]
indeed, looks fine now!
